modules/rostests/apitests/win32nt/ntuser/NtUserSetTimer.c

```diff
@@ -71,9 +72,6 @@ static BOOL test1(void)
     ULONGLONG startTime;
     MSG msg = { NULL };
 
-    int minMessages = ((float)SLEEP_TIME / (float)TEST1_INTERVAL) * (1 - TIME_TOLERANCE);
-    int maxMessages = ((float)SLEEP_TIME / (float)TEST1_INTERVAL) * (1 + TIME_TOLERANCE);
```

Comment:
I do wonder why that old code didn't trigger any compiler warning about "implicit cast and potential loss of data"? Do we have the warning level reduced here artificially?

Reply:
When I thought about it, my old time calculation didn't make sense because messages can arrive independently of SLEEP_TIME. And it's more of a coincidence that it roughly matched the times in Windows. The tests don't have to be rigorous, it's not a benchmark, but a test of whether all messages arrive at all in any reasonable time. The times will be different in emulators and different in real environments, on different architectures, etc.
That's why the interval should be wide.

Reply:
If you were the author of those tests yourself and changed your mind, that would be absolutely ok ofc. I browsed the log for a while, but couldn't spot the original author quickly. If it was you, then feel free to pimp them to your hearts desire. I just don't want any potential historic wisdom/insight getting lost.

Reply:
Hey, don't worry, I'm the author. We're not losing any awesome code, just code that I've found doesn't work very well.